docs/docsite/rst/community/communication.rst

```diff
@@ -130,27 +83,24 @@ Working groups
 
 Working Groups are a way for Ansible community members to self-organize around particular topics of interest.
 
-Our community working groups are represented in `Matrix rooms <https://matrix.to/#/#space:ansible.com>`_ and  `Forum groups <https://forum.ansible.com/g>`_.
+Our community working groups are represented in `Forum groups <https://forum.ansible.com/g>`_ and additionally in `Matrix rooms <https://matrix.to/#/#space:ansible.com>`_.
 See those links for a complete list of communications channels.
 
-Many of them meet in chat. If you want to get involved in a working group, join the Matrix room or IRC channel where it meets or comment on the agenda.
-
-
 Forming a new working group
```

Comment:
I guess 99% of people aren't interested in how to setup a Working Group.
Should we move this elsewhere, do we have an existing page (or Forum Guide) on how to setup a Working Group?

Reply:
The forum 'new group' button seems to have most of the details and is a form template we can update if we want to add more details - https://forum.ansible.com/g/custom/new

Reply:
Agreed, that info is imo a bit extra, reorganized, PTAL